deployment/docker-stack-wait.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/sh
```

Comment:
sorry for the dumb questions, what's this file for?

Reply:
The source repo for this file is https://github.com/sudo-bmitch/docker-stack-wait.

It's a script that waits (with a timeout) until a `docker stack deploy` operation has finished, since the command itself is non-blocking. Before using this script, if something went wrong with the deploy, the github runner wouldn't actually show a failure. Now it does!

I used to just download the script from the source as part of the action, but that's not great for security. Hence I just stick it in our repo now.